tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,277 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// e2e implements the e2e tests.
+package e2e_test
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
```

Comment:
From https://pkg.go.dev/io/ioutil:
`As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.`